src/CachePoolTest.php

```diff
@@ -520,7 +540,11 @@ public function testGetItemsInvalidKeys($key)
             return;
         }
 
-        $this->cache->getItems(['key1', $key, 'key2']);
+        $items = $this->cache->getItems(['key1', $key, 'key2']);
+
+        if ($items instanceof \Traversable) {
+            iterator_to_array($items);
```

Comment:
This is wrong as this should never be reached. The interface expects ``getItems`` to throw an exception. You are changing the behavior of the tested API here as the exception is not in the same place anymore.